Makefile

```diff
@@ -38,6 +38,33 @@ check_in_container: test_image
 		-v $(CURDIR)/files/packit-service.yaml:/root/.config/packit-service.yaml \
 		$(TEST_IMAGE) make check
 
+# This is my target so don't touch it! :) How to:
```

Comment:
?

Reply:
so... the target for running tests in a container is not good enough for me and I keep changing it all the time when working on p-s: so I decided to start my own target (which everyone can use)

Reply:
Oh, it understood that as "don't touch it and don't try to use it"..;)